Gruntfile.js

```diff
@@ -28,6 +28,13 @@ module.exports = function(grunt) {
           fix: true
         },
         src: ['src/**/*.js', 'test/tests/**/*.js']
+      },
```

Comment:
looks good approach. 
btw
There are some common config fields in both `source` and `sourceNofix`, I wonder if we don't have to write them twice?

Reply:
@montoyamoraga  @therewasaguy  do you think we can do smth better here ?